.github/stale.yml

```diff
@@ -3,7 +3,10 @@ daysUntilStale: 120
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
 # Issues with these labels will never be considered stale
-exemptLabels: false
+exemptLabels:
+  - "do not close"
+  - "feature request"
+  - "scope:doc"
```

Comment:
I would add also `"new feature"`, which is used for planning new feature that has to be implemented